test-runtime/Cargo.toml

```diff
@@ -9,7 +9,7 @@ sp-runtime = "5.0.0"
 codec = { package = "parity-scale-codec", version = "2", default-features = false, features = ["derive", "full", "bit-vec"] }
 
 [build-dependencies]
-subxt = { path = "../subxt", version = "0.16.0" }
```

Comment:
let's remove the `version` entirely from this to save us a little future trouble.